ckanext/datagovuk/plugin.py

```diff
@@ -315,11 +315,19 @@ def get_helpers(self):
 
     # IMiddleware
 
+    # Ignore these data errors on sentry as these would already be reported to publishers to resolve or raise
+    IGNORED_DATA_ERRORS = [
+        "Found more than one dataset with the same guid",   # DCat
+        "Errors found for object with GUID",                # Spatial
+        "Job timeout:",                                     # Harvest
+        "was aborted or timed out",                         # Harvest
+        "Too many consecutive retries for object"           # Harvest
+    ]
+
     def before_send(self, event, hint):
-        # disable sentry while CKAN is processing objects which have timed out due to upgrade
-        return None
-        # return None if [i for i in ['localhost', 'integration', 'staging'] if i in config.get('ckan.site_url')] \
-        #     else event
+        return None if [i for i in ['localhost', 'integration', 'staging'] if i in config.get('ckan.site_url')] or \
```

Comment:
Could you explain in the commit why we're allowing errors in Production here?

(Or better still, just add 'production' to this list for now, and then remove it from the list in a later commit),

Reply:
Thats the intention, we want sentry errors from Production in this commit